configure

```diff
@@ -76,6 +78,24 @@ OPTIONS = {
     "ncurses": "if_available"
 }
 
+def download_windows_depencies():
+    if not os.path.isdir("download"):
+        os.mkdir("download")
+    
+    # dowload the last release zip
+    zip_name = "openage-dep-x64-windows.zip"
+    zip_path = os.path.join("download", zip_name)
+    link = "https://github.com/SFTtech/openage-dependencies/releases/latest/download/openage-dep-x64-windows.zip"
+    file = open(zip_path, "wb")
+    file.write(requests.get(link).content)
```

Comment:
This requires that we add `requests` as a dependency just for Windows, which I don't think is a good idea. I would suggest that we don't include this method in configure and just provide the powershell script.